lib/sdam/topology_description.js

```diff
@@ -155,15 +156,15 @@ class TopologyDescription {
         maxSetVersion,
         maxElectionId,
         commonWireVersion,
-        this.options
+        options
       );
     }
 
     if (topologyType === TopologyType.Unknown) {
-      if (serverType === ServerType.Standalone) {
+      if (serverType === ServerType.Standalone && options.directConnection !== false) {
```

Comment:
This looks like a logic error to me, if I'm able to pass in a valid connection string with `directConnection` (which requires a single seed), then I should not be in a situation where I'm `Unknown` and a `Standalone` server comes in. Can you point me to the test that requires this change?